packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emrcontainers/start-job-run.ts

```diff
@@ -295,8 +306,15 @@ export class EmrContainersStartJobRun extends sfn.TaskStateBase implements iam.I
 
     return jobExecutionRole;
   }
-  private grantMonitoringPolicies() {
 
+  private jobExecutionRoleFromArn(exectionRoleArn?: string): iam.IRole | undefined {
+    if (exectionRoleArn === undefined) {
+      return undefined;
+    }
+    return iam.Role.fromRoleArn(this, 'Job-Execution-Role-From-Arn', exectionRoleArn);
+  }
```

Comment:
As I understand it, this PR is implemented to solve a problem with `executionRole`, because `iam.Role` can't be used here. In this code snippet, `executionRoleArn` is used to create an `iam.Role` again. Isn't it enough to note in the `executionRole` property that e.g. importedValues cannot be used? Do we really need this new property?